recipe/meta.yaml

```diff
@@ -24,21 +24,31 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - make
+    - make   # [not win]
     - pkg-config
-    - gperf
+    - gperf  # [not win]
+    - m2-gperf  # [win]
     - expat
-    - gettext 
-    - libtool
-    - automake
-    - autoconf
-    - patch
+    - gettext   # [not win]
+    - libtool   # [not win]
+    - automake  # [not win]
+    - autoconf  # [not win]
+    - patch     # [not win]
+    - m2-patch  # [win]
+    - python  # [win]
+    - meson   # [win]
+    - ninja   # [win]
 
   host:
     - freetype
     - libxml2
```

Comment:
It seems like usage of libxml2 is interrelated with expat if there is **no** `SAX1` support in `libxml2` https://cgit.freedesktop.org/fontconfig/tree/configure.ac#n445 otherwise `libxml2` is not required https://cgit.freedesktop.org/fontconfig/tree/configure.ac#n425? 
I see that out libxml2 doesn't enable SAX1 explicitly https://github.com/AnacondaRecipes/libxml2-feedstock/blob/master/recipe/build.sh#L11-L14 (https://github.com/ConradIrwin/libxml2/blob/b368604d9ba04fec68dbc93733e1c2737766747e/configure.in#L165).
That means that libxml2 is required. Or am I wrong?

Reply:
I assume so ... therefore it is in host.  I haven't touched this, as @danpetry had introduced it prior.

Reply:
I didn't touch it either, it was here already when I worked on it